guild/ipy.py

```diff
@@ -64,6 +44,27 @@
 
 from guild.commands import runs_impl
 
+# ipy makes use of the full Guild API and so, like main_bootstrap,
```

Comment:
These lines shuffled to clear up a flake8 issue about imports needing to be up top.